app/kudos/templates/shared/kudos_card_profile.html

```diff
@@ -17,41 +17,69 @@
+                <img src="/dynamic/avatar/{{ kudos_group.list.0.username }}" alt="{{ kudos_group.list.0.from_username }}" class="rounded-circle" width="30" height="30">
+              {% endfor %}
+            </div>
+          </div>
+        </div>
+
+        <div class="content-hover flip-card__hold flip-card__back">
+          {% include 'shared/kudos_card_hover_content.html' with relation='to' %}
+        </div>
       </div>
     </div>
   </div>
   {% endfor %}
 {% else %}
-  {% for kt in kudos|slice:":8" %}
+  {% regroup kudos|dictsort:"kudos_token_cloned_from.name" by kudos_token_cloned_from.name as kudos_dupe %}
+  {% for kudos_group in kudos_dupe|slice:":8" %}
   <div class="col-xs-12 col-sm-6 col-lg-3 mb-3">
-    <div class="kd-card kd-extended" aria-haspopup="true" tabindex="0">
-      <a href="/kudos/{{ kt.kudos_token_cloned_from.id }}/{{ kt.kudos_token_cloned_from.name }}" class="" title="{{ kt.kudos_token_cloned_from.name|humanize_name }}" tabindex="0">
-        <img src="{% static kt.kudos_token_cloned_from.image %}" alt="{{ kt.kudos_token_cloned_from.name|humanize_name }}" class="img-thumbnail border-transparent kd-shadow" width="250">
-      </a>
-      <div class="kd-content">
-        <div class="kd-title">{{ kt.kudos_token_cloned_from.name|humanize_name }}</div>
-        {{ kt.kudos_token_cloned_from.description|truncatechars:60 }}
-      </div>
-      <div class="content-hover">
-        <a class="kd-title" href="/kudos/{{ kt.kudos_token_cloned_from.id }}/{{ kt.kudos_token_cloned_from.name }}">{{ kt.kudos_token_cloned_from.name|humanize_name }}</a>
+    <div class="kd-card kd-extended flip-card" aria-haspopup="true" tabindex="0">
+      <div class="flip-card__flipper">
+        <div class="flip-card__hold flip-card__front">
+          <img src="{% static kudos_group.list.0.kudos_token_cloned_from.image %}" alt="{{ kudos_group.list.0.kudos_token_cloned_from.name|humanize_name }}" class="img-thumbnail border-transparent kd-shadow" width="250">
+          <div class="kd-content">
+            <div>
+              <a href="/kudos/{{ kudos_group.list.0.kudos_token_cloned_from.id }}/{{ kudos_group.list.0.kudos_token_cloned_from.name }}" class="kd-title" title="{% trans 'Go to ' %}{{ kudos_group.list.0.kudos_token_cloned_from.name|humanize_name }}" tabindex="0">
+                {{ kudos_group.list.0.kudos_token_cloned_from.name|humanize_name }}
+              </a>
+            </div>
+            <p>
+              {{ kudos_group.list.0.kudos_token_cloned_from.description|truncatechars:60 }}
+            </p>
+            <div class="flip-card__stack-img">
+              {% if kudos_group.list|length > 5 %}<b>+{{kudos_group.list|length|add:"-5"}}</b>{% endif %}
+              {% for kt in kudos_group.list|slice:":5" %}
+                <img src="/dynamic/avatar/{{ kudos_group.list.0.from_username }}" alt="{{ kudos_group.list.0.from_username }}" class="rounded-circle" width="30" height="30">
```

Comment:
@mbeacom we are moving away from this right ?

Reply:
Yes - We should be using the avatar template filter `load avatar_tags` -> `{% avatar_url <profile> %}` - So if you have access to the actual profile, this would be ideal: `KudosTransfer.sender_profile` should do it.

Reply:
Even if you opt to not do that for the time being, we should migrate these to use `{% url 'url_name' %}` vs hardcoded routes.